static/src/stylesheets/module/content/_gallery.head.scss

```diff
@@ -1,22 +1,68 @@
 .content__labels--gallery,
-.l-side-margins--gallery .content__meta-container,
 .content__headline--gallery,
-.content__standfirst--gallery {
+.gallery__header {
     @include mq(desktop) {
         margin-left: gs-span(3) + $gs-gutter;
         max-width: gs-span(8);
     }
 }
 
-.l-side-margins--gallery {
-    .content__meta-container {
-        position: relative;
-        width: auto;
+.gallery__meta-container {
+    margin-bottom: $gs-baseline;
+
+    &:before {
+        content: '';
+        border-top: 1px solid $media-mute;
+        padding-top: $gs-baseline / 4;
+        display: block;
+        width: gs-span(1);
+    }
+
+    .byline,
+    .content__dateline {
+        border: 0;
+        min-height: 0;
+        padding: 0;
+    }
+}
+
+.meta__extras--gallery {
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px dotted $media-mute;
```

Comment:
There is still lots of tonal stuff in this file which is messy, but I would rather think about that as another piece of work